backend/generator.js

```diff
@@ -4,11 +4,11 @@ var mailer = require('./mailer');
 var uuidV4 = require("uuid/v4");
 var validation = require("../public/scripts/validation.js");
 var spawn = require('child_process').spawn;
-var output = require('../util/output');
+var socketHandler = require('./socketHandler.js');
 
-exports.executeScript = function (socket, formData) {
+exports.executeScript = function (socket, formData, cb) {
```

Comment:
I suppose `cb` stands for `callback`? If that is the case, consider naming it `callback`. shouldn't be a lot of extra keystrokes.